package.json

```diff
@@ -50,6 +50,7 @@
     "dependencies": {
         "@apify/http-request": "^1.1.5",
         "@apify/ps-tree": "^1.1.3",
+        "@types/mocha": "^5.2.7",
```

Comment:
I reckon this is just a forgotten line? We don't use `mocha`.

Reply:
That is true. I am not sure why the original `tsconfig.json` "referenced" or "declared" types dependency on `mocha` I'll remove this.